spec/controllers/carts_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe CartsController, type: :controller do
+  describe "GET #index" do
+    it "render show" do
+      get :index, params: {locale: I18n.locale}
+      expect(response).to render_template(:index)
```

Comment:
Sao it là render show mà expect lại index cậu

check giá trị các biến instance `@` nữa hì